sirepo/template/srw.py

```diff
@@ -367,6 +370,36 @@ def _extract_coherent_modes(model, out_info):
     return out_file
 
 
+def _get_recent(data_filename, backup_data_filename):
+    if os.path.getmtime(backup_data_filename) > os.path.getmtime(data_filename):
+        return PKDict(
+            recent=backup_data_filename,
+            old=data_filename,
+        )
+    return PKDict(
+        recent=data_filename,
+        old=backup_data_filename,
+    )
+
+
+def _count_lines(text):
```

Comment:
It will be more efficient (and sufficient) to check for the size of the files. py.path has a size check so should be easy.